src/database/models/roommodel.js

```diff
@@ -0,0 +1,19 @@
+module.exports = (sequelize, DataTypes) => {
+  const roommodel = sequelize.define('roommodel',{
```

Comment:
Please follow the model naming convention; if a single word should start with an upper case, if composite, UpperCamelCase, and singular. Preferably call your model Room or RoomModel, but remember the model generates the table and the table is for Rooms, so I'd suggest you just call your model Room

Reply:
Thanks , I am going to work on it